src/tools/lifter/selftest_arm8.log

```diff
@@ -3,37 +3,37 @@ MANUAL TESTS - ARMv8
 
 8B020020 (add x0, x1, x2) @ 0x10030 - OK
  []
-|- bir_is_lifted_inst_prog arm8_bmr (Imm64 65584w) (WI_end 0w 16777216w)
```

Comment:
Oh boy, did this change two times now? from k12 to k13 and then back from k13 to k14? Haha. At least I have this déjà vu feeling.

Reply:
Yes, I think you are correct.